.github/workflows/build_binary.yml

```diff
@@ -34,10 +34,13 @@ jobs:
           zip relay-Linux-x86_64-debug.zip relay.debug
           mv relay relay-Linux-x86_64
 
-      - uses: actions/upload-artifact@v3
+      - uses: actions/upload-artifact@v4
         with:
-          name: ${{ github.sha }}
+          name: artifact-linux
           path: target/release/relay-Linux-x86_64*
+          if-no-files-found: 'error'
+          # since this artifact will be merged, compression is not necessary
+          compression-level: '0'
```

Comment:
Is there a downside to keeping the default compression? I'd assume compressed it uploads/downloads faster.

Reply:
Network inside GHA is quite fast, binaries tend to be harder to compress.

Here's a test run with default compression: https://github.com/getsentry/relay/actions/runs/10926922965

upload-artifact runs take 1, 2, 4s uncompressed, 4, 5, 5s compressed.
merge takes a full 30s as it has to decompress.

Reply:
Thanks for testing, nice!